### PR TITLE
remove App.Srv().Store.Channel() reference

### PR DESCRIPTION
**Pull request by isacikgoz**
#### Summary
We are experimenting the idea of removing `*app.Server` access from the `app.App`. The idea is make every access pass through app layer and isolate server. For the first step, I took converting `app.Srv().Store.Channel()` use cases to app methods. This PR can be considered as an example for this effort.

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -4614,6 +4618,10 @@
     "id": "app.channel.update_last_viewed_at_post.app_error",
     "translation": "Unable to mark channel as unread."
   },
+  {
+    "id": "app.channel.update_member.app_error",
+    "translation": ""
+  },
```
Comment (agnivade) [inline i18n/en.json:4624]: Hmm .. this is probably incorrect. I wouldn't expect translations to change/ new ones to be added. Can you check this?

Comment (isacikgoz) [reply i18n/en.json]: oops, yeah. Checked and saw that I missed the unexported `app.updateChannelMember` method. Thanks for the heads up!

**Review by agnivade — APPROVED** [review]
Good stuff!

**Comment by isacikgoz** [comment]
Outdated, closing.

**Pull request closed without merging**